src/python/pants/testutil/rule_runner.py

```diff
@@ -6,6 +6,7 @@
 import atexit
 import dataclasses
 import functools
+import logging as logginglib
```

Comment:
Sadly we have a function called `logging` below, so we must alias to `logginglib`.